Node-DC-EIS-cluster/server-cluster.js

```diff
@@ -20,7 +20,8 @@ var appConfig = require('./config/configuration');
 var app_host = appConfig.app_host;
 var port = appConfig.app_port;
 const os = require('os');
-var cpuCount = appConfig.cpu_count;
+var cpu_count = appConfig.cpu_count;
+var cpuCount = Number(cpu_count);
```

Comment:
no need to create both, please refactor to
var cpuCount = Number(appConfig.cpu_count);